sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/RecognizeCustomFormsLiveTests.cs

```diff
@@ -500,28 +506,18 @@ public async Task StartRecognizeCustomFormsWithoutLabelsCanParseBlankPage()
         }
 
         [RecordedTest]
-        [TestCase(true)]
-        [TestCase(false, Ignore = "https://github.com/Azure/azure-sdk-for-net/issues/12319")]
-        public async Task StartRecognizeCustomFormsWithoutLabelsCanParseMultipageFormWithBlankPage(bool useStream)
+        public async Task StartRecognizeCustomFormsWithoutLabelsCanParseMultipageFormWithBlankPage()
```

Comment:
This test was failing because of the same issue we had in the test above. It's been years since we started ignoring it.

In this case, instead of splitting it into two tests, I'm just removing the failing scenario since the error is already being validated by another test.